testdata/expected/docker/python-no-url/python-no-uri/PipPackage.graph.json

```diff
@@ -1,84 +0,0 @@
-{
```

Comment:
Oh actually, got it confused with the non-docker output. If this testcase is omitted, that means the toolchain will not properly run inside the docker container (described in the `Dockerfile`). That is currently the way that toolchains are run on Sourcegraph.com, so this would fail on Sourcegraph. If it's a matter of just updating the `Dockerfile` to reflect the other changes made to the non-Docker install process, I can help with that.